securedrop/tests/migrations/migration_b58139cfdc8c.py

```diff
@@ -206,6 +225,18 @@ def check_downgrade(self):
                 except NoSuchColumnError:
                     pass
 
+            journalists_sql = "SELECT * FROM journalists"
+            journalists = db.engine.execute(text(journalists_sql)).fetchall()
+            for journalist in journalists:
+                try:
```

Comment:
Looking at these makes me realize I wrote my test for the `checksum` column wrong. All three of these `NoSuchColumnError` checks should be:

```python
with pytest.raises(NoSuchColumnError):
    row['column_name']
```

This will catch the case where the column exists. The two you added have the edge case of passing when those columns are `None`, and the one I wrote is even more wrong of catching none of the cases where that column exists.